CITATION.cff

```diff
@@ -0,0 +1,40 @@
+# YAML 1.2
+# Metadata for citation of this software according to the CFF format (https://citation-file-format.github.io/)
+cff-version: 1.0.3
+message: If you use this software, please cite it using these metadata.
+title: cf_xarray
+doi: 10.5281/zenodo.4749736
+repository-code: https://github.com/xarray-contrib/cf-xarray
+license: Apache-2.0
+version: 0.5.2
+date-released: 2021-05-11
+keywords:
+- cf-conventions
+- xarray
+- metadata
+
+authors:
+- affiliation: National Center for Atmospheric Research, USA
+  family-names: Cherian
+  given-names: Deepak
+  orcid: https://orcid.org/0000-0002-6861-8734
+- affiliation: National Oceanography Center, Southampton
+  family-names: Almansi
+  given-names: Mattia
+  orcid: https://orcid.org/0000-0001-6849-3647
+- affiliation: Ouranos, Inc.
+  family-names: Bourgault
+  given-names: Pascal
+  orcid: https://orcid.org/0000-0003-1192-0403
+- affiliation: National Center for Atmospheric Research, USA
+  family-names: Kent
+  given-names: Julia
+- family-names: Magin
+  given-names: Justus
```

Comment:
@keewis can you provide an affiliation and orcid? Only if you want to...

Reply:
I don't have any of those, yet...